alerts/kubelet.libsonnet

```diff
@@ -52,20 +58,26 @@
             // Some node has a capacity of 1 like AWS's Fargate and only exists while a pod is running on it.
             // We have to ignore this special node in the KubeletTooManyPods alert.
             expr: |||
-              count by(%(clusterLabel)s, node) (
-                (kube_pod_status_phase{%(kubeStateMetricsSelector)s,phase="Running"} == 1) * on(instance,pod,namespace,%(clusterLabel)s) group_left(node) topk by(instance,pod,namespace,%(clusterLabel)s) (1, kube_pod_info{%(kubeStateMetricsSelector)s})
+              count by (%(clusterLabel)s, node) (
+                (kube_pod_status_phase{%(kubeStateMetricsSelector)s, phase="Running"} == 1)
+                * on (%(clusterLabel)s, namespace, pod) group_left (node)
+                group by (%(clusterLabel)s, namespace, pod, node) (
+                  kube_pod_info{%(kubeStateMetricsSelector)s}
+                )
```

Comment:
Here is the "More robust de-dupe of pod count in KubeletTooManyPods alert".

Reply:
btw this change is covered by an existing test:

https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/03c13f9b38a0162309b987637bd905805e305b00/tests.yaml#L406-L435